src/components/tools/Charts/DotAndWhisker/DotAndWhisker.svelte

```diff
@@ -5,6 +5,9 @@
 
   export let data;
 
+  let seriesVisibility;
```

Comment:
I don't think you need to declare the variable here because doing `$: seriesVisibility = ...` below declares it.

Reply:
True. It's so easy to forget coding when you take too long a break :-)

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/176/commits/c5504427d2ed17fc52d4580a7f002c551e371f3a